README.md

```diff
@@ -79,306 +79,273 @@ int main()
-simply use `BEMAN_EXEMPLAR_BUILD_TESTING=OFF` as documented in upstream [CMake documentation](https://cmake.org/cmake/help/latest/module/CTest.html):
+<summary> Details of CMake arguments. </summary>
 
-```shell
-cmake -B build -S . -DBEMAN_EXEMPLAR_BUILD_TESTING=OFF
+#### `BEMAN_EXEMPLAR_BUILD_TESTS`
+
+Enable building tests and test infrastructure. Default: ON.
+Values: { ON, OFF }.
+
+You can configure the project to have this option turned off via:
+
+```bash
+cmake -B build -S . -DCMAKE_CXX_STANDARD=20 -DBEMAN_EXEMPLAR_BUILD_TESTS=OFF
 ```
 
+> [!TIP]
+> Because this project requires Google Tests as part of its development
+> dependency,
+> disable building tests avoids the project from pulling Google Tests from
+> GitHub.
+
+#### `BEMAN_EXEMPLAR_BUILD_EXAMPLES`
+
+Enable building examples. Default: ON. Values: { ON, OFF }.
+
 </details>
 
 ## Integrate beman.exemplar into your project
 
-<details>
-<summary> Use beman.exemplar directly from C++ </summary>
-<!-- TODO Darius: rewrite section!-->
-
-If you want to use `beman.exemplar` from your project,
-you can include `beman/exemplar/*.hpp`  files from your C++ source files
+To use `beman.exemplar` in your C++ project,
+you should include relevant headers from `beman.exemplar` in your source files.
 
-```cpp
+```c++
 #include <beman/exemplar/identity.hpp>
 ```
 
-and directly link with `libbeman.exemplar.a`
+You will then link your project to `beman.exemplar`.
 
-```shell
-# Assume /opt/beman.exemplar staging directory.
-$ c++ -o identity_usage examples/identity_usage.cpp \
-    -I /opt/beman.exemplar/include/ \
-    -L/opt/beman.exemplar/lib/ -lbeman.exemplar
+### Produce beman.exemplar static library locally
+
+You can include exemplar's headers locally
+by producing a static `libbeman.exemplar.a` library.
+
+```bash
+cmake --workflow --preset gcc-release
+cmake --install build/gcc-release --prefix /opt/beman.exemplar
 ```
 
-</details>
+This will generate such project structure at `/opt/beman.exemplar`.
 
-<details>
-<summary> Use beman.exemplar directly from CMake </summary>
+```txt
+/opt/beman.exemplar
+├── include
+│   └── beman
+│       └── exemplar
+│           └── identity.hpp
+└── lib
+    └── libbeman.exemplar.a
+```
 
-<!-- TODO Darius: rewrite section! Add examples. -->
+### Linking your project to beman.exemplar with CMake
 
-For CMake based projects, you will need to use the `beman.exemplar` CMake module to define the `beman::exemplar` CMake target:
+For CMake based projects,
+you will need to use the `beman.exemplar` CMake module
+to define the `beman::exemplar` CMake target:
 
 ```cmake
 find_package(beman.exemplar REQUIRED)
 ```
 
-You will also need to add `beman::exemplar`
-to the link libraries of any libraries or executables that include `beman/exemplar/*.hpp` in their source or header file.
+You will also need to add `beman::exemplar` to the link libraries of
+any libraries or executables that include beman.exemplar's header file.
```

Comment:
Looks good.

Note that it would be a little tricky to get the pluralization of "header file" versus "header files" correct if a project template contained this text. Some projects would be one header like `beman.exemplar`, but many would not.